llvm/lib/Target/RISCV/GISel/RISCVInstructionSelector.cpp

```diff
@@ -271,37 +270,25 @@ const TargetRegisterClass *RISCVInstructionSelector::getRegClassForTypeOnBank(
 bool RISCVInstructionSelector::selectCopy(MachineInstr &MI,
                                           MachineRegisterInfo &MRI) const {
   Register DstReg = MI.getOperand(0).getReg();
-  Register SrcReg = MI.getOperand(1).getReg();
 
-  if (Register::isPhysicalRegister(SrcReg) &&
-      Register::isPhysicalRegister(DstReg))
+  if (DstReg.isPhysical())
```

Comment:
Are the select functions supposed to return `true` if a change was made?

Reply:
I believe they are supposed to return true for success and false for failure.